src/components/ASPerformance.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PlotlyGraph from './Plot.js';
+import Select from 'react-select';
+import Highlighter from 'react-highlight-words';
+import 'react-select/dist/react-select.css';
+import '../css/temp.css' //this is temp import - needs to be removed for bundle
+
+import { fetchAsDataIfNeeded, AsIsSelected } from '../actions/ASactions';
+
+export class ASPerformance extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      plotlyData: [],
+      graphOptions: {
+        legend: {x:0, y:1},
+        height: 200,
+        margin: {
+          l: 30,r: 30,
+          b: 30,t: 0
+        },
+        font: {
+          size: 9,
+          color: '#7f7f7f'
+        }
+      }
+    }
+  }
+
+
+  computeState(props=this.props) {
+    let lineColors = [
+      'rgb(214, 39, 40)', 'rgb(31, 119, 180)', 'rgb(44, 160, 44)',
+      'rgb(255, 127, 14)', 'rgb(238, 130, 238)'
+    ]
```

Comment:
why not to include this colors in state initially.  Eg: included in  graphOptions